pkg/resolution/common/errors.go

```diff
@@ -51,77 +51,77 @@ func NewError(reason string, err error) *Error {
 }
 
 var (
-	// ErrorRequestInProgress is a sentinel value to indicate that
+	// ErrRequestInProgress is a sentinel value to indicate that
 	// a resource request is still in progress.
-	ErrorRequestInProgress = NewError("RequestInProgress", errors.New("Resource request is still in-progress"))
+	ErrRequestInProgress = NewError("RequestInProgress", errors.New("Resource request is still in-progress"))
```

Comment:
In this PR I think we should include backwards compatibility for anything related to remote resolution, since this could be used by clients / resolver implementations. Unfortunately I think this means we're going to need similar changes for everything under `remote` and `resolution` dirs :(

Trusted resources we can break since this is alpha. We should include a breaking change notice though (which we categorize differently than action required in release notes 🤷 ).

Reply:
I think I've now got this aligned with your comments; PTAL.